guides/v2.2/architecture/archi_perspectives/components/modules/mod_depend.md

```diff
@@ -31,13 +31,45 @@ A module with a *hard dependency* on another module cannot function without the
 * Deserializes an object declared in another module.
 * Uses or modifies the database tables used by another module.
 
+Module hard dependency defines in the `require` section of `app/code/<Vendor>/<Module>/composer.json` file. For example, :
```

Comment:
Perhaps like this:

```suggestion
The `require` section of `app/code/<Vendor>/<Module>/composer.json` file contains hard dependency definitions for the module. For example, :
```

Reply:
@hguthrie , thank you!

**implemented**